back/spec/acceptance/cosponsors_initiatives_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource 'CosponsorsInitiatives' do
+  explanation 'Cosponsorship of initiatives (proposals) by users'
+
+  before do
+    header 'Content-Type', 'application/json'
+    user = create(:user)
+    @cosponsors_initiative = create(:cosponsors_initiative, user: user, initiative: create(:initiative))
+    header_token_for user
+  end
+
+  patch '/web_api/v1/cosponsors_initiatives/:id/accept_invite' do
+    let(:id) { @cosponsors_initiative.id }
+
+    example_request 'Accept an invitation to cosponsor an initiative' do
+      assert_status 200
+    end
```

Comment:
```suggestion
    example 'Accept an invitation to cosponsor an initiative' do
      expect do
        do_request
      end.to change { @cosponsors_initiative.reload.status }.from('pending').to('accepted')
      assert_status 200
    end
```
This additional small check could be useful too.